### PR TITLE
Add support for marking Reader posts seen/unseen

### PR DESCRIPTION
**Pull request by ScoutHarris**
### Description

Ref https://github.com/wordpress-mobile/WordPress-iOS/issues/15355
WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/15638

This adds functionality to toggle a post's `is_seen` status.

### Testing Details

Can be tested with the referenced WPiOS PR.

- [ ] Please check here if your pull request includes additional test coverage.

### CODE REVIEW

File: WordPressKit/ReaderPostServiceRemote+V2.swift [hunk 1]
```diff
@@ -44,4 +44,43 @@ extension ReaderPostServiceRemote {
...
+        let path = self.path(forEndpoint: endpoint, withVersion: ._2_0)
+        
+        let params = [
+            "feed_id": feedID,
+            "feed_item_ids": [feedItemID],
+            "source": "reader-ios"
+        ] as [String: AnyObject]
+        
+        wordPressComRestApi.POST(path, parameters: params, success: { (responseObject, httpResponse) in
+            success()
```
Comment (aerych) [inline WordPressKit/ReaderPostServiceRemote+V2.swift:79]: Probably we should check what's returned in the response object.  It should be a dictionary with a single key `status`.  The value of status can be true or false depending on if the call succeeded.  Looks like there are a few (unlikely) scenarios where false could be returned.

**Review by aerych — CHANGES_REQUESTED** [review]
Hola @ScoutHarris 👋 
I had just one nitpick looking at the code. 
Back to you!

File: WordPressKit/ReaderPostServiceRemote+V2.swift (hunk 1 above)
Comment (ScoutHarris) [reply WordPressKit/ReaderPostServiceRemote+V2.swift:79]: Good catch. And done!

**Review by aerych — APPROVED** [review]
LGTM! :shipit:

**Comment by ScoutHarris** [comment]
Thanks @aerych !

**Pull request merged**